db/migrate/20230713093406_create_users.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.integer :posts_comment
```

Comment:
- [ ] As requested in the project requirements, column names should match the ERD diagram. Please make sure to change this column name to posts_counter` as on the ERD diagram.